### DISCUSSION TITLE
Outlet type for Power Feed

### DISCUSSION THREAD

**Discussion by davidabuckner** [netbox-community/netbox#10243] [Ideas]
Under the Power Feed there should be an outlet type field to assign the proper outlet type to the feed.  This can then be used to connect a Power Plug from a device.

**Comment by cyberndj** [comment]
I like this idea too. You should consider making this an issue since it will help model the connections/receptacles that the feeds from panels use to connect. ie, is a feed's connection  a NEMA 15R, L6-30R, etc. I would think you could re-use the current plug types.

**Comment by swagnerug** [comment]
The idea has some appeal, however since feeds are coming from panels and (at least in my mind/deployment ) represent a breaker and a circuit that may have multiple receptacles of different types  and simultaneously be feeding multiple devices. I'm not sure it makes sense unless modeling each receptacle on the wall (which could be a very valid use case for some places) is something you want to do but then you get into quad boxes versus double outlets etc.
